cool-core/src/test/java/com/nus/cool/core/refactor/ProcessorTest.java

```diff
@@ -56,7 +56,12 @@ public void processQueryAndValidResult(String queryDir) throws IOException {
     this.coolModel = new CoolModel(this.cubeRepo);
     coolModel.reload(cohortProcessor.getDataSource());
     CubeRS cube = coolModel.getCube(cohortProcessor.getDataSource());
+
+    // get current dir path
+    File currentVersion = this.coolModel.loadLatestVersion(cohortProcessor.getDataSource());
+    // cohortProcessor.readExistingCohort("../CubeRepo/health_raw/v00000012");
```

Comment:
commented-out codes.

Reply:
Add some tests to it, and bring it back.   We currently don't have any filed indicating where and how to read the previous cohort result in the query file.